src/assistant.rs

```diff
@@ -33,55 +39,59 @@ pub struct OpenAIAssistant {
     instructions: String,
     debug: bool,
     api_key: String,
+    version: OpenAIAssistantVersion,
 }
 
 impl OpenAIAssistant {
     //Constructor
     pub async fn new(model: OpenAIModels, open_ai_key: &str, debug: bool) -> Result<Self> {
```

Comment:
@darioalessandro, now there is no change to the constructor so this should be backwards compatible.

The constructor defaults to V1, so everyone that uses it should still be good.

The constructor no longer needs to be async, but I left it that way to preserve backwards compatibility.